desktop/core/src/desktop/js/utils/url/changeURL.ts

```diff
@@ -45,7 +43,11 @@ const changeURL = (newURL: string, params?: Record<string, string | number | boo
   } else if (window.location.hash) {
     url += window.location.hash;
   }
-  window.history.pushState(null, '', url);
+  if (isReplace) {
```

Comment:
replace/push state only if new url and old url is different.

Reply:
Make sense, Let me add the condition.